lnbits/helpers.py

```diff
@@ -58,6 +58,9 @@ def template_renderer(additional_folders: Optional[List] = None) -> Jinja2Templa
     t.env.globals["LNBITS_DENOMINATION"] = settings.lnbits_denomination
     t.env.globals["SITE_TAGLINE"] = settings.lnbits_site_tagline
     t.env.globals["SITE_DESCRIPTION"] = settings.lnbits_site_description
+    t.env.globals["LNIBITS_HP_ELEMENTS"] = settings.lnbits_hp_elements
+    t.env.globals["LNIBITS_CUSTOM_BADGE"] = settings.lnbits_custom_badge
+    t.env.globals["LNIBITS_CUSTOM_BADGE_COLOR"] = settings.lnbits_custom_badge_color
```

Comment:
All these three properties have a typo (`LNIBITS` -> `LNBITS`).
```suggestion
    t.env.globals["LNBITS_HP_ELEMENTS"] = settings.lnbits_hp_elements
    t.env.globals["LNBITS_CUSTOM_BADGE"] = settings.lnbits_custom_badge
    t.env.globals["LNBITS_CUSTOM_BADGE_COLOR"] = settings.lnbits_custom_badge_color
```